### ISSUE TITLE
Train and benchmark several RL algorithms

### ISSUE THREAD

**Issue by michaelwm** [uwreact/uwreact_robot#45]
# 🚀 Feature Request

Part of #42. Depends on #44. Once an environment is set up, it will be easy to train several of the RL algorithms provided by `keras-rl`. All of these algorithms should be bench marked and a team discussion take place on which one to use for production training. The computation library for performing these tasks will be `tensorflow` as it is easy to deploy on production cloud services. The focus is on 2019: creating a generic tool for this is not essential, but it will be very beneficial for future years, and the task of a separate issue.

**Comment by michaelwm** [comment]
Yes, `keras-rl` lets us run `tensorflow` which is never going anywhere, and is also pretty damn fast. So it gives us speed and security.

**Comment by michaelwm** [comment]
We are now going to be using `pytorch` for several reasons, after talking to industry experts on machine learning. The main reasons are:
1. *Faster development speed.* The number one reason highlighted by every expert, regardless of whether they preferred `pytorch` or not, is that `pytorch` has undoubtedly the fastest development cycle of any machine learning library.
2. *Brighter future*: `pytorch` is growing faster than `tensorflow`/`keras` and is expected to continue. It's development community is, at the moment, growing faster as well, and with it comes exceptional libraries and support.
